pkg/didcomm/common/service/did_comm_msg.go

```diff
@@ -132,6 +135,21 @@ func (m DIDCommMsgMap) Type() string {
 	return res
 }
 
+// ParentThreadID returns the message parentT hreadID
+func (m DIDCommMsgMap) ParentThreadID() string {
```

Comment:
I see code duplication with [1252](https://github.com/hyperledger/aries-framework-go/pull/1252)

can this function be extracted into a helper function?

Reply:
It is duplicated because this PR contains code from the previous PR. You will not see these changes after the first one will be merged and this one rebased.